src/scss/06_components/_tags.scss

```diff
@@ -0,0 +1,53 @@
+$tag-gap: $spacing-half;
+
+.content-tags__tags {
+  @include flex-gap($tag-gap);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-bottom: 0;
+  margin-top: 0;
+  padding-bottom: ($spacing-and-half - $tag-gap);
+  padding-left: 0;
+  padding-top: $spacing-and-half;
+}
+
+.content-tags__tags__tag {
+  list-style: none;
+  margin-bottom: $tag-gap;
+  margin-top: 0;
+
+  a,
+  span {
+    @include font('small');
+    display: inline-block;
+    line-height: 12px;
+    text-decoration: none;
+    transition: 0.2s ease-in-out all;
+  }
+}
+
+.content-tags__tags--interactive a {
+  background-color: transparent;
+  border: 2px solid $color-black;
+  border-radius: $spacing-and-half;
+  padding: $spacing 20px;
+
+  &:hover {
+    background-color: $color-black;
+    border-color: $color-black;
+    color: $color-white;
+  }
+
+  &:focus {
+    border-color: $color-black;
+    outline: 2px solid $color-black;
+    outline-offset: 2px;
+  }
+}
+
+.content-tags__tags--static span {
+  background-color: $color-silver;
+  border-radius: $spacing;
+  padding: $spacing-half $spacing;
```

Comment:
Wait a minute, `$spacing-half + line-height + $spacing-half = 8px + 12px + 8px = 28px ≠ 32px`

The small tag was supposed to be 32px high, right? Now it's only 28px. So after removing the `line-height: 12px`, the actual line-height for font-small is 24px, we should have `4px + 24px + 4px = 32px` So get rid of the `line-height: 12px` and set top and bottom padding on small tags to be 4px.

To me it seems that you have tried to get it to 32px, but something has gone wrong. As you've set border-radius to $spacing = 16px which is half of 32px as it should be.